### ISSUE TITLE
Info on future releases

### ISSUE THREAD

**Issue by bartvde** [pivotal-cf/pivotal-ui#315]
I see you guys moved from v2.1.0-alpha.2 to v3.0.0-alpha.0 recently, why is this? And what are the releases we can expect in the near future? Wondering if I should follow the 2.1 alpha path or the 3.0 alpha path? Any guidance? TIA.

**Comment by atomanyih** [comment]
We just upgraded to be compatible with react 0.14. Technically, this is a
breaking change because we don't support 0.13 any more

I mainly made it a 3 alpha so that people wouldn't auto upgrade by accident
and get borked

We're still figuring out our plan for the next real release though, so it's
still in flux. Either way, new work will probably only be on one path (for
now, 3 alpha)

If you have any suggestions for how to make this less confusing as a
consumer we'd love to hear them!

**Comment by bartvde** [comment]
I was confused since for 2.1.0 alpha2 package.json files it already stated react 0.14 (https://github.com/pivotal-cf/pivotal-ui/blob/v2.1.0-alpha.2/package.json)
Right, my guess was also that v3 would be for react 0.14 support.

Thanks for clearing this up. It's clear to me now.

**Comment by bartvde** [comment]
I just tried 3.0.0 alpha0 but run into this with node 4.2.2 any ideas? :

```
Error: Cannot find module 'react-bootstrap/lib/ModalHeader' from '/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/pui-react-modals'
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: write after end
    at writeAfterEnd (/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/watchify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:198:12)
    at DestroyableTransform.Writable.write (/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/watchify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:243:5)
    at DestroyableTransform.Writable.end (/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/watchify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:474:10)
    at Readable.<anonymous> (/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/watchify/bin/cmd.js:47:16)
    at emitOne (events.js:82:20)
    at Readable.emit (events.js:169:7)
    at Labeled.<anonymous> (/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/watchify/node_modules/browserify/node_modules/read-only-stream/index.js:28:44)
    at emitOne (events.js:77:13)
    at Labeled.emit (events.js:169:7)
    at Labeled.<anonymous> (/Users/bartvandeneijnden/opengeo/git/sdk/node_modules/watchify/node_modules/browserify/node_modules/labeled-stream-splicer/node_modules/stream-splicer/index.js:130:18)
```

**Comment by bartvde** [comment]
So is pui-react-modals missing a dependency?

```
Barts-MacBook-Air:sdk bartvandeneijnden$ find . -name "react-bootstrap"
./node_modules/pui-react-alerts/node_modules/react-bootstrap
./node_modules/pui-react-dropdowns/node_modules/react-bootstrap
./node_modules/pui-react-grids/node_modules/react-bootstrap
./node_modules/pui-react-tabs/node_modules/react-bootstrap
```

**Comment by atomanyih** [comment]
Ah probably. Good catch

**Comment by atomanyih** [comment]
closing this because the missing dependency is unrelated to original post

**Comment by atomanyih** [comment]
fixed it tho: 375a8c9ccebe2ccc8a52b9893be987a4a1547a23

**Comment by bartvde** [comment]
thanks for fixing!
